launch-checklist.md

```diff
@@ -1 +1,20 @@
+# Replicache Launch Checklist
 
+Before you launch with Replicache in your product, it's a good idea to double-check that you have correctly covered all the small details of integration. This list can help you determine if you might have missed a detail, or deferred and then forgotten about it.
+
+- **JS SDK**
+  - if you wish to version the schema of the mutations in `Push` and the `clientView` returned in `Pull`, version information is provided in `ReplicacheOptions.schemaVersion`
+  - re-auth is handled for `Push` and `Pull` via `getPushAuth` and `getPullAuth`
+- **All endpoints**
+  - the user is authenticated and the `clientID` verified to belong to that user
+- **Push endpoint**
+  - the `lastMutationID` for a client is updated transactionally along with the pushed mutations' effects
```

Comment:
We need to create a *Server Requirements* doc that details exactly what is needed from server at a minimum. Like for example, I believe that CosmosDB's "consistent prefix" setting is consistent enough for Replicache's purposes.

Reply:
Agree we should create such a doc. In the meantime, I've included the content inline.

Reply:
Also PTAL at the "It is extremely important" item re consistency.